app/policies/application_policy.rb

```diff
@@ -42,6 +42,8 @@ def rails_admin?(action)
     case action
       when :dashboard
         user.is_admin?
+      when :import
```

Comment:
Is there an `import` action for rails_admin? If not, we don't need to add this policy action, since it will never be checked.

Reply:
I was wondering if we could bulk add import students to other things instead of just trips in the future